File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        p=0
        n=0
        for i in nums:
            if i<0:
                n=n+1
            else:
                if i>0:
                    p=p+1
        if p>n:
            return p
        else:
            return n

File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int p=0,n=0;
        for (int num:nums)
        {
            if(num>0)
            {
                p=p+1;
            }
            else
            { 
                if(num<0)
                {
                    n=n+1;
                }
            }
        }
        if(p>n)
        {
            return p;
        }
        else
        {
            return n;
        }
    }
}